tools/spdx-check.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# List of files to check, excluding vendor directories
+files=$(git diff --name-only --diff-filter=A origin/master..HEAD | grep -v "vendor/")
```

Comment:
I wonder if `origin/master..HEAD` is going to work for backports (once this script becomes part of some LTS branch).
Maybe we could pass `${{ github.event.pull_request.base.ref }}` as an input argument to this script (or `github.base_ref`, I don't know which one is the correct in this case).

Reply:
Yep, good suggestion! I'll change to `base.ref`

Reply:
Looks like he fixed version works